src/Microsoft.Data.SqlClient/netfx/src/Microsoft/Data/SqlClient/SqlInternalConnectionSmi.cs

```diff
@@ -162,22 +162,6 @@ override internal bool IsLockedForBulkCopy
             }
         }
 
-        override internal bool Is2000
-        {
-            get
-            {
-                return false;   // Can't be direct connecting to 2000.
-            }
-        }
-
-        override internal bool Is2005OrNewer
-        {
-            get
-            {
-                return true;    // Must be direct connecting to 2005 or newer.
-            }
-        }
-
         override internal bool Is2008OrNewer
```

Comment:
Not removing this?

Reply:
Preferably not in this PR. The last major release only barred connections to SQL 2000; it's still technically possible (just very far out of support) to connect to SQL 2005 without the library throwing an exception.